CONTRIBUTING.md

```
@@ -1 +1,135 @@
+## Pull Request Process
+
+1. **Create a new branch** from the `main` branch.
+
+2. **Make your changes** in the new branch.
+
+3. **Add tests or examples** to cover your changes.
+
+4. **Update the documentation** if necessary.
+
+5. **Create a changeset** (optional):
+	```bash
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)